docs/docs/dev-guide-usb-logging.md

```diff
@@ -42,12 +45,26 @@ CONFIG_USB_UART_DTR_WAIT=n
 
 ## Viewing Logs
 
-After flashing the updated ZMK image, the board should expose a USB CDC ACM device, that you can connect to and view the logs.
+After flashing the updated ZMK image, the board should expose a USB CDC ACM device that you can connect to and view the logs.
 
-On Linux, this should be a device like `/dev/ttyACM0` and you can connect with `minicom` or `tio` as usual, e.g.:
+<Tabs
+defaultValue="linux"
+values={[
+{label: 'Linux', value: 'linux'},
+{label: 'Windows', value: 'win'},
+]}>
+<TabItem value="linux">
 
+On Linux, this should be a device like `/dev/ttyACM0` and you can connect with `minicom` or `tio` as usual, e.g.:
 ```
 sudo tio /dev/ttyACM0
 ```
+</TabItem>
+<TabItem value="win">
+
+On Windows, you can use the Arduino IDE which contains a built-in Serial Monitor. Download and install it from [their website](https://www.arduino.cc/en/main/software), then connect your board and under Tools select "Serial Monitor".
```

Comment:
Can we also add a note about PuTTY being an option, which is a much lighter weight choice?

Reply:
I'll try that out, I happened to have Arduino from setting up BlueMicro and wasn't sure what alternative options there were.